docs/pages/server-access/guides/vscode.mdx

```diff
@@ -12,7 +12,7 @@ This guide explains how to use Teleport and Visual Studio Code's remote SSH exte
 
 - OpenSSH client.
 - Visual Studio Code with the [Remote - SSH extension](https://code.visualstudio.com/docs/remote/ssh#_system-requirements)
-for the Remote - SSH extension.
+for the Remote - SSH extension. The setting `remote.SSH.useLocalServer` should be set to `false`.
```

Comment:
We include this instruction in Step 2 as well—should we remove this usage since it's more prominent in Step 2?

Reply:
removed